src/EPR.RegulatorService.Facade.API/Controllers/RegulatorController.cs

```diff
@@ -171,13 +182,13 @@ public async Task<IActionResult> GetUserDetails()
             }
             else
             {
-                _logger.LogError("Failed to fetch the organisations list for the user {userId}", userId);
+                _logger.LogInformation("Failed to fetch the organisations list for the user {userId}", userId);
```

Comment:
this is was previously _logger.LogError any reson why it is now _logger.LogInformation?